bom/runtime/pom.xml

```diff
@@ -74,7 +74,7 @@
         <maven-plugin-annotations.version>3.5.2</maven-plugin-annotations.version>
         <plexus-component-annotations.version>1.7.1</plexus-component-annotations.version>
         <!-- What we actually depend on for the annotations, as latest Graal is not available in Maven fast enough: -->
-        <graal-sdk.version>19.2.1</graal-sdk.version>
+        <graal-sdk.version>19.3.0</graal-sdk.version>
```

Comment:
Apparently they changed the groupId of the artifacts. At least that was in the announcement.

Reply:
Fixed. I didn't see that, thanks for the info!